docs/basics.md

```diff
@@ -123,6 +124,13 @@ within('.form-signup-home', function () {
 I.see('There were problems creating your account.');
 ```
 
+iframe example
+```js
+within({frame:["#editor"]},function(){
```

Comment:
why not just `frame: '#editor'`, what is the difference on using arrays and single arguments?
I think both options should be provided and documented